rosdep/python.yaml

```
@@ -6460,6 +6460,10 @@ python3-gnupg:
   opensuse: [python3-python-gnupg]
   ubuntu: [python3-gnupg]
 python3-google-cloud-texttospeech-pip: *migrate_eol_2025_04_30_python3_google_cloud_texttospeech_pip
+python3-gpiozero:
+  debian: [python3-gpiozero]
+  fedora: [python3-gpiozero]
+  ubuntu: [python3-gpiozero]
```

Per the check failure, python3-gpiozero does not appear to be available in Ubuntu Bionic.

```suggestion
  ubuntu:
    '*': [python3-gpiozero]
    bionic: null
```